src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Color getRandomColor() {
+        return Color.values()[new Random().nextInt(Color.values().length)];
```

Comment:
move random to the class field, like this example:
![image](https://github.com/mate-academy/jv-lottery/assets/12864128/62d4d371-453d-4877-9fc5-b245d08218ca)

Reply:
Ok ,  Int.Idea.  suggests it final...  
it is better to made random FINAL to?